README.md

```diff
@@ -103,21 +124,50 @@ mkdir -p $HOME/.local/bin && cp lite-xl $HOME/.local/bin/
 mkdir -p $HOME/.local/share/lite-xl && cp -r data/* $HOME/.local/share/lite-xl/
 ```
 
+#### Add Lite XL to PATH
+
 If `$HOME/.local/bin` is not in PATH:
 
 ```sh
 echo -e 'export PATH=$PATH:$HOME/.local/bin' >> $HOME/.bashrc
 ```
 
-To get the icon to show up in app launcher:
+> **Note**
+> Some systems might not load `.bashrc` when logging in.
+> This can cause problems with launching applications from the desktop / menu.
+
+#### Add Lite XL to application launchers
+
+To get the icon to show up in app launcher, you need to create a desktop
+entry and put it into `/usr/share/applications` or `~/.local/share/applications`.
+
+Here is an example of the desktop entry, assuming Lite XL is in PATH:
+
+```ini
+# com.lite_xl.LiteXL.desktop
```

Comment:
Is this comment needed?
I guess this is about the file name, so it might be better to put this clearly above.